metadata.rb

```diff
@@ -6,8 +6,7 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '1.0.0'
 depends          "application_ruby", "~> 4.0"
-depends          "apt",              "~> 4.0"
-depends          "git",              "~> 4.6"
-depends          'nginx_passenger',  "~> 0.5.7"
```

Comment:
I don't believe we do. I think the dependency was supposed to be removed and wasn't as a mistake. We use nginx but we install the package manually and build the template file ourself, and we don't use passenger, eliminating the need for the nginx_passenger cookbook.